test/readdlm.jl

```diff
@@ -203,3 +203,7 @@ end
     reshape(Any[1,2000.1,Float64(22222222222222222222222222222222222222),true,0x3,false,10e6,-10.34], 2, 4), Any)
 
 @test isequaldlm(readcsv(IOBuffer("-9223355253176920979,9223355253176920979"), Int64), Int64[-9223355253176920979  9223355253176920979], Int64)
+
+# directory path is supplied by the user instead of a file path (#11485)
+dir = mktempdir()
```

Comment:
apparently appveyor is not happy with this use of `mktempdir` - would also be best practice to clean up after the test is finished